posix_time.h

```diff
@@ -0,0 +1,11 @@
+#ifndef __POSIX_TIME_H__
+#define __POSIX_TIME_H__
+
+#include <ctime>
+#include <winsock2.h>
```

Comment:
This is where timeval is defined under Windows. Linux version of time.h defines this structure. We actually need it.